### PR TITLE
Cram5

### PR DESCRIPTION
**Pull request by seba90**

### CODE REVIEW

File: cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp [hunk 1]
```diff
@@ -135,6 +135,9 @@
 (defun export-log-to-owl (filename)
   (send-prolog-query-1 (create-query "rdf_save" (list (concatenate 'string "\\'/home/ros/user_data/" filename "\\'" ) "[graph(\\'LoggingGraph\\')]"))))
 
+(defun export-belief-state-to-owl (filename)
+  (json-prolog:prolog-simple-1 (concatenate 'string "rdf_save(" (concatenate 'string "'/home/ros/user_data/" filename "'" "," "[graph('belief-state')])"))))
```
Comment (gaya-) [inline cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp]: Could you please give a default filename for the function?

(defun blabla (&optional (filename "blabla"))

File: cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp [hunk 2]
```diff
@@ -132,9 +132,12 @@
 (defun send-cram-set-perception-result (action-inst res)
   (send-prolog-query-1 (create-query "cram_set_perception_result" (list action-inst res))))
 
-(defun export-log-to-owl (filename)
+(defun export-log-to-owl (&optional (filename (concatenate 'string "default_log_" (write-to-string (truncate (cram-utilities:current-timestamp))) ".owl")))
   (send-prolog-query-1 (create-query "rdf_save" (list (concatenate 'string "\\'/home/ros/user_data/" filename "\\'" ) "[graph(\\'LoggingGraph\\')]"))))
 
+(defun export-belief-state-to-owl (&optional (filename (concatenate 'string "default_belief_state_" (write-to-string (truncate (cram-utilities:current-timestamp))) ".owl")))
+  (json-prolog:prolog-simple-1 (concatenate 'string "rdf_save(" (concatenate 'string "'/home/ros/user_data/" filename "'" "," "[graph('belief-state')])"))))
```
Comment (gaya-) [inline cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp]: Hey, could you please use "/home/ease/logs" as the directory? There is no "ros/user_data" directory on the robot, cuz this belief-state is stored locally.

Comment (gaya-) [reply cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp]: @seba90

Comment (seba90) [reply cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp]: I fixed the store path for the belief state

File: cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp [hunk 3]
```diff
@@ -160,12 +163,12 @@
 
 (defun send-object-action-parameter (action-inst object-designator)
   (let ((object-instance-id (write-to-string (desig::desig-prop-value object-designator :NAME))))
-    (send-rdf-query (convert-to-prolog-str action-inst) "knowrob:object" (convert-to-prolog-str object-instance-id))
+    (send-rdf-query (convert-to-prolog-str action-inst) "knowrob:objectActedOn" (convert-to-prolog-str object-instance-id))
     object-instance-id))
 
 (defun get-object-name (object-name)
```
Comment (gaya-) [inline cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp]: @seba90 Hey Sebastian, you're not using this function anywhere. Where was it supposed to be used?

Comment (gaya-) [reply cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp]: @seba90 Ah, found the place, actually right here on L162. Ok, it's an easy fix, I'll do it, so never mind.

**Pull request merged**